README.rst

```diff
@@ -1,16 +1,32 @@
 ecl-bundles
 ===========
 
-This repository has been created as a 'testing ground' for new ECL bundle development.
-Bundles that are accepted into this repository have passed initial review and testing
-by HPCC Systems team, but are ultimately maintained by the original authors.
+This repository serves as a central list of all known ECL bundles.
 
-Individual bundles that have passed the initial testing phase have been moved to the
-https://github.com/ecl-bundles site. These can be installed individually directly from
-the ecl-bundle tool. To use a bundle from this repository, it is necessary to clone
-the whole repository first, then install the individual bundle file or directory using
-the ecl bundle tool.
+Bundles listed here fall into three categories: 
+ - supported bundles are maintained by the HPCC platform team, and tested against each
+   release.
+ - approved bundles are maintained by external contributors, but have passed initial review
+   and testing by the HPCC platform team and thus should be considered stable and safe to
+   use, and should remain compatible across platform versions
+ - other bundles are created and maintained by external contributors, and have not been
+   approved by the platform team. Use at your own risk
+
+If you have developed a bundle that you would like to add to any of these lists, create
+a pull request against this file to add your bundle's repository to the appropriate list.
 
 For more information about how to create an Ecl bundle, see the `Ecl Bundle Writer's Guide`_.
 
 .. _`Ecl Bundle Writer's Guide`: https://github.com/hpcc-systems/HPCC-Platform/blob/master/ecl/ecl-bundle/BUNDLES.rst
+
+Supported bundles
+=================
+PerformanceTesting - https://github.com/hpcc-systems/PerformanceTesting
+
+Approved bundles
+================
+MySqlImport - https://github.com/hpcc-systems/MySqlImport
```

Comment:
@ghalliday Arguably this should link to https://hithub.com/ghalliday/MySqlImport (if that existed) and/or we should move this to the Supported list